www/src/utils/sidebar/get-active-item-parents.js

```diff
@@ -15,7 +15,7 @@ const isParentActive = (sections, parentTitle) => {
 const getActiveItemParents = (itemList, activeItemLink, activeItemParents) => {
   if (activeItemLink.parentTitle) {
     const bar = isParentActive(itemList, activeItemLink.parentTitle)
-    activeItemParents.push(bar.title)
```

Comment:
This gets tricky to update the Tutorials section, which is a bit of a unique case. Currently the top-level item says "Tutorial (step-by-step)" but it's a parent to both Additional Tutorials and the upcoming Themes Tutorials, so this title doesn't make sense to be a breadcrumb parent for those. But in the context of the main tutorial where this heading shows in the sidebar, it does make sense. 

I think the best solution would be to add a `breadcrumbTitle` or something to optionally alter the breadcrumb display since even though we could split on ` (` and remove the latter part, it still isn't quite right and mucking with pluralization of a single word seems like a bad way to go. I wanted to add my comment here since I tried updating it myself to move this forward, but had to switch gears to something else for the themes launch. So we can get to this when we get to it!

Reply:
Alright I can get to this later then. My first solution actually just pulled the top level route: `/tutorial/part-one` => `tutorial` or `/docs/guides/plugins` => `docs` but maybe that wouldn't be the best solution either. Do you mean add a breadcrumbTitle to the sidebar links yaml files?

As far as comments go in those util functions, I can add some context there too.

Reply:
Ah yes, that's probably thanks to our evolving (and confusing) IA! 😅 

Yeah, I meant a value in the tutorial-links yaml file. I should have clarified! This isn't a huge rush, I just wanted to comment while it was fresh in my mind. Thanks for your work on it!

Reply:
I just pushed an update for this. It was pretty simple and works for those top level edge cases where the hierarchy might be a little weird.